testsuite/openmodelica/cruntime/debugDumps/testDumpEvents.mos

```diff
@@ -44,262 +44,198 @@ simulate(bbTestDump, stopTime=3.0, simflags="-lv LOG_EVENTS_V");
 // LOG_EVENTS        | info    | status of zero crossings at time=0
 // |                 | |       | | [1] (pre:  0) -1 = h <= 0.0 and v <= 0.0
 // |                 | |       | | [2] (pre:  0) -1 = h <= 0.0
-// |                 | |       | | [3] (pre:  0)  1 = v <= 0.0
```

Comment:
Why does this disappear? We compiled the model and it seems that removing the preOptModule `removeEqualFunctionCalls` does this. 

Although it seems like it is more correct this way since `v<=0.0` is not really a zero crossing to begin with, but still i do not get how this is affected.

Reply:
I noticed the same but didn't investigate it further.

Reply:
I would say thats a problem in `removeEqualFunctionCalls` and should be analysed in a different ticket / PR

Reply:
We could do that, but if the coverage test works fine, we should rather remove `removeEqualFunctionCalls` completely.